code/modules/clothing/suits/wiz_robe.dm

```diff
@@ -180,4 +180,48 @@
 	src.robe_charge = FALSE
 	sleep(30)
 	src.robe_charge = TRUE
-	usr << "<span class='notice'>\The robe hums, its internal magic supply restored.</span>"
\ No newline at end of file
+	usr << "<span class='notice'>\The robe hums, its internal magic supply restored.</span>"
+
+
+//Shielded Armour
+
+/obj/item/clothing/suit/space/hardsuit/shielded/wizard
+	name = "battlemage armour"
+	desc = "Not all wizards are afraid of getting up close and personal."
+	icon_state = "battlemage"
+	item_state = "battlemage"
+	recharge_rate = 1
+	current_charges = 15
+	recharge_cooldown = INFINITY
+	shield_state = "shield-red"
+	shield_on = "shield-red"
+	min_cold_protection_temperature = ARMOR_MIN_TEMP_PROTECT

```

Comment:
@Cheridan @AnturK 

This one isn't space proof, it just uses the hardsuit path to keep all the shield code in one place.